tests/utils.js

```diff
@@ -248,7 +248,7 @@ const revertDb = (except, ignoreRefresh) => {
 const deleteUsers = async (users, meta = false) => {
   const usernames = users.map(user => `org.couchdb.user:${user.username}`);
   const userDocs = await request({ path: '/_users/_all_docs', method: 'POST', body: { keys: usernames } });
-  const medicDocs = await request({ path: '/medic/_all_docs', method: 'POST', body: { keys: usernames } });
+  const medicDocs = await request({ path: `/${constants.DB_NAME}/_all_docs`, method: 'POST', body: { keys: usernames}});
```

Comment:
These changes aren't technically needed since we remap /medic/ to the "right" url inside api. However, it's inline with how we talk about URLs in the rest of our test suite so it made more sense to me.